### ISSUE TITLE
How to make a selection like .isin() in pandas?

### ISSUE THREAD

**Issue by jackyckma** [vaexio/vaex#375]
It seems like a typical use case. Shall I use vaex.register_function dectorator to add this functionality? or it is indeed possible with current API?

**Comment by jackyckma** [comment]
just settled with this mapping and select approach:

```
selected_items = {'itemA': 1, 'itemB': 1}
df['select'] = df.item_key.map(selected_items, default_value=0, allow_missing=True)
df.y_value.max(selection=[None, df.select==0, df.select==1])
```

**Comment by maartenbreddels** [comment]
`isin()` is not supported yet, but a must-have, I'll keep this open till then.

**Comment by JovanVeljanoski** [comment]
Closed via #441
